rust/test/willbe/tests.rs

```diff
@@ -0,0 +1,70 @@
+use std::path::PathBuf;
+
+use willbe::prelude::*;
+
+#[ test ]
+fn package_from_path()
+{
+  // current package is a package
+  let path = PathBuf::from( "" );
+  let package = Package::try_from( path.to_owned() );
+
+  assert!( package.is_ok() );
+  assert_eq!( path, *package.unwrap().path() );
+}
+
+#[ test ]
+fn workspace_from_path()
+{
+  // current package is a package, not a workspace
+  assert!( Workspace::try_from( PathBuf::from( "" ) ).is_err() );
+  // main workspace is a workspace
+  assert!( Workspace::try_from( PathBuf::from( "../../../" ) ).is_ok() );
```

Comment:
Tests should use test assets instead of real modules. 
Please, see the [tests of module `publisher`](https://github.com/Wandalen/wTools/tree/alpha/rust/test/publisher) as an example.